packages/rrweb/src/replay/index.ts

```diff
@@ -684,6 +691,7 @@ export class Replayer {
     if (this.config.UNSAFE_replayCanvas) {
       this.preloadAllImages();
     }
+    this.disableInteract();
```

Comment:
Just to be sure, why do we need to add this here as well? Was this an oversight before?

Reply:
I moved it here so that it is called when the iframe is ready, otherwise the event listener does not end up getting attached.